arm-compute/quickstart-templates/aliases.json

```
@@ -16,34 +16,69 @@
             "version": "latest",
             "architecture": "x64"
           },
+          "CentOS85Gen2":  {
+            "publisher":  "OpenLogic",
+            "offer":  "CentOS",
+            "sku":  "8_5-gen2",
+            "version":  "latest",
+            "architecture": "x64"
+          },
           "Debian": {
             "publisher": "Debian",
             "offer": "debian-10",
             "sku": "10",
             "version": "latest",
             "architecture": "x64"
           },
+          "Debian11":  {
+            "publisher":  "Debian",
+             "offer":  "debian-11",
+             "sku":  "11-backports-gen2",
+             "version":  "latest",
+            "architecture": "x64"
+          },
           "Flatcar": {
             "publisher": "kinvolk",
             "offer": "flatcar-container-linux-free",
             "sku": "stable",
             "version": "latest",
             "architecture": "x64"
           },
+          "FlatcarLinuxFreeGen2":  {
+            "publisher":  "kinvolk",
+             "offer":  "flatcar-container-linux-free",
+             "sku":  "stable-gen2",
+             "version":  "latest",
+            "architecture": "x64"
+          }
```

Add a comma to prevent parsing errors.
```suggestion
          },
```